crates/uv-extract/src/stream.rs

```diff
@@ -172,7 +188,7 @@ async fn untar_in(
                 let mode = file.header().mode()?;
                 let has_any_executable_bit = mode & 0o111;
                 if has_any_executable_bit != 0 {
-                    if let Some(path) = crate::tar::unpacked_at(dst, &file.path()?) {
```

Comment:
We can get rid of this whole vendored `unpacked_at` implementation. It turns out the crate now returns the target path.